### PR TITLE
Update en.yml

### PR DESCRIPTION
**Pull request by Dan13L3**
@baarkerlounger Hopefully captured all the person vs. tenant distinctions here

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -154,30 +154,30 @@ en:
         question_required: "You must answer whether the person is still serving in the UK armed forces as you told us they're a current or former regular"
         question_not_required: "You cannot answer whether the person is still serving in the UK armed forces as you told us they're not a current or former regular"
       preg_occ:
-        no_female: "You must answer 'no' as there are no female tenants aged 16-50 in the property"
+        no_female: "You must answer 'no' as there are no females aged 16-50 in the household"
```
Comment (baarkerlounger) [inline config/locales/en.yml:157]: `female persons`? `women`? `females` reads a bit odd...

Comment (Dan13L3) [reply config/locales/en.yml:157]: @baarkerlounger We've tried to go with male/female to keep it consistent with the way we phrase the gender identity question, but I think 'women' works fine too, so happy to update

Comment (baarkerlounger) [reply config/locales/en.yml:157]: @Dan13L3 your call, want to change it (if so just push another commit) or want me to merge this as is?

Comment (Dan13L3) [reply config/locales/en.yml:157]: @baarkerlounger Perhaps merge as is, as I can do a general review of the gender questions, as they might need making consistent throughout

**Pull request merged**